code/M3D/M3DWrapper.h

```diff
@@ -53,8 +53,8 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <string>
 
 // Assimp specific M3D configuration. Comment out these defines to remove functionality
-#define ASSIMP_USE_M3D_READFILECB
-#define M3D_ASCII
+//#define ASSIMP_USE_M3D_READFILECB
```

Comment:
It should be possible to set these from CMake without editing the source

Reply:
I don't think so. If they are really as hazardous as you claim them to be, then better if there's no option in Assimp to enable them.